static/app/views/alerts/create.spec.tsx

```diff
@@ -359,6 +364,8 @@ describe('ProjectAlertsCreate', function () {
         );
         expect(metric.startSpan).toHaveBeenCalledWith({name: 'saveAlertRule'});
 
+        expect(mockUpdateOnboardingTasks).toHaveBeenCalledTimes(1);
```

Comment:
this make sure that when an issue alert is created and the demo mode is not active, we mark the alert rule task as complete